### DISCUSSION TITLE
Can I import the output result into a txt file? How to do it?

### DISCUSSION THREAD

**Discussion by Zhao-shenghao** [github/securitylab#164] [General] [answered]
Can I import the output result into a txt or else file? How to do it?
![1](https://user-images.githubusercontent.com/43957169/89011985-f2eb5500-d343-11ea-9041-0727b067c846.png)

**Comment by rdmarsh2** [comment, marked as answer]
There isn't currently a way to do that through VS Code directly (but there's a related open issue: https://github.com/github/vscode-codeql/issues/439). What you can do currently is look through the CodeQL extension log in the output pane, and find the last call to `bqrs interpret`, which will have paths to a JSON-formatted SARIF file (ending in .sarif) and a binary query results file (ending in .bqrs). The SARIF file is human readable, and extensions to view it directly in the problems view are available for VS Code and Visual Studio. You can also use `codeql bqrs interpret` to convert a bqrs file into a CSV, but that won't have path information included.
